### PR TITLE
ROX-11443: Change default value for roxctl image scan --include-snoozed

### PR DESCRIPTION
**Pull request by mtodor**
## Description

We are changing the default option for snoozed CVEs in order to match UI behaviour, where also snoozed CVEs are filtered by default.

This PR contains:
- change the default to `false`
- add a test for the default value
- add an entry into the changelog

## Checklist
- [x] Investigated and inspected CI test results
- [x] Unit test and regression tests added
- [x] Evaluated and added CHANGELOG entry if required
- ~[ ] Determined and documented upgrade steps~ - no need for upgrade steps
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~ - should not be relevant

## Testing Performed

1. Spin up local stackrox stack, make a port forward, and export password to `ROX_PASSWORD` environment variable
2. Run the following command:
```
./bin/darwin/roxctl --insecure-skip-tls-verify --insecure --endpoint https://localhost:8000 -p "${ROX_PASSWORD}" image scan --image debian:10.1
```
3. After that, log in to UI. Open: https://localhost:8000
4. Go to: `https://localhost:8000/main/vulnerability-management`
5. Click on `99 CVEs` next to `docker.io/library/debian:10.1`
6. Click on one CVE from the list and select `DEFER AND APPROVE` to snooze that CVE
7. If you go to: `https://localhost:8000/main/vulnerability-management` - you should see one CVE less for `docker.io/library/debian:10.1`
8. Run the following command:
```
./bin/darwin/roxctl --insecure-skip-tls-verify --insecure --endpoint https://localhost:8000 -p "${ROX_PASSWORD}" image scan --image debian:10.1
```
The number of CVEs should be the same as in UI.
9. Run the following command:
```
./bin/darwin/roxctl --insecure-skip-tls-verify --insecure --endpoint https://localhost:8000 -p "${ROX_PASSWORD}" image scan --image debian:10.1 --include-snoozed
```
The number of CVEs should be bigger by 1 because all CVEs are included.

### CODE REVIEW

**Review by stehessel — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 ## [NEXT RELEASE]
 
+- ROX-11443: The default value for `--include-snoozed` option of `roxctl image scan` command is set to `false`.
```
Comment (dhaus67) [inline CHANGELOG.md:9]: Maybe we should also mention the behavior change, i.e. what the impact is of setting it to `false`?

File: roxctl/image/scan/scan_test.go [hunk 2]
```diff
@@ -588,3 +588,11 @@ func (s *imageScanTestSuite) runLegacyOutputTests(cases map[string]outputFormatT
 		})
 	}
 }
+
+func (s *imageScanTestSuite) TestScan_IncludeSnoozed() {
+	s.Run("disabled by default", func() {
```
Comment (dhaus67) [inline roxctl/image/scan/scan_test.go:593]: nit: no need to use `s.Run()` here, you don't need to create a sub-test for a single test case.

**Review by dhaus67 — APPROVED** [review]

**Review by rukletsov — APPROVED** [review]
LGTM pending open comments and a happy CI.

**Review by dhaus67 — APPROVED** [review]
LGTM let's pray to the CI gods 🙏

**Pull request merged**